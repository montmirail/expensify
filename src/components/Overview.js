import React from 'react';

// ----- components -----
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const Overview = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default Overview;