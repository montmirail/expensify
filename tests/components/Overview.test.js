import React from 'react';
import { shallow } from 'enzyme';
import Overview from '../../src/components/Overview';

test('should render overview correctly', () => {
    const wrapper = shallow(<Overview />);
    expect(wrapper).toMatchSnapshot();
});