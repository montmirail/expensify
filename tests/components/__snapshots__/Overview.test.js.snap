// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render overview correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
